learning_journal_basic1/views/default.py

```diff
@@ -0,0 +1,43 @@
+from pyramid.response import Response
+from pyramid.view import view_config
+
+from sqlalchemy.exc import DBAPIError
+
+from ..models import MyModel
+
+
+@view_config(route_name='home', renderer='../templates/list.jinja2')
+def my_view(request):
+    try:
+        query = request.dbsession.query(MyModel)
+        one = query.filter(MyModel.title == 'one').first()
```

Comment:
If this is supposed to be sending a list of your Learning Journal entries to your front page, then why are you only retrieving one model instance?